### PR TITLE
Ignore RELATED-TO when entry links to itself

### LINKED ISSUE #14: Ignore RELATED-TO when entry links to itself

### PR DESCRIPTION
**Pull request by patrickunterwegs**
Closes #14

### CODE REVIEW

File: src/main/java/at/bitfire/ical4android/JtxICalObject.kt [hunk 1]
```diff
@@ -1211,8 +1211,10 @@ duration?.let(props::add)
             )
         }
 
-
         this.relatedTo.forEach { related ->
+            if(related.text == this.uid)                // ignoring entries that are linked to itself (issue with Tasks.org)
+                return@forEach
+
```
Comment (abaker) [inline src/main/java/at/bitfire/ical4android/JtxICalObject.kt:1217]: I haven't heard about this before, do you know the steps to reproduce this in Tasks.org?

**Comment by rfc2822** [comment]
Marking as draft because it's not clear whether this is required

File: src/main/java/at/bitfire/ical4android/JtxICalObject.kt (hunk 1 above)
Comment (patrickunterwegs) [reply src/main/java/at/bitfire/ical4android/JtxICalObject.kt:1217]: Hello @abaker , there was an issue that was hard to reproduce. This was an attempt to resolve the problem. However, I found another place where the actual problem originates, so nothing to worry for Tasks.org for now ;-) I will delete this PR

**Comment by patrickunterwegs** [comment]
Found another place where a fix is needed so that this change is not necessary.

**Pull request closed without merging**